security/advancedtls/crl.go

```diff
@@ -120,7 +121,7 @@ func x509NameHash(r pkix.RDNSequence) string {
 	for _, canonRdn := range r {
 		b, err := asn1.Marshal(canonRdn)
 		if err != nil {
-			log.Print(err)
+			grpclog.Print(err)
```

Comment:
`Error` instead of `Print`?

Reply:
yes, that's better, we would not expect a failure here